examples/immutable-values/immutable_values.bal

```diff
@@ -0,0 +1,75 @@
+import ballerina/io;
+
+type Department record {
```

Comment:
Shall we add comments for these as well. Just a small note to explain what is happening here.